kolibri/core/assets/src/views/SideNav.vue

```diff
@@ -397,7 +397,7 @@
     },
     $trs: {
       navigationLabel: {
-        message: 'Main user navigation',
+        message: 'Main user menu',
```

Comment:
Is this string possibly being pulled from other places? I can still see the old wording 😕 

![Ubuntu18 (start)  Running  - Oracle VM VirtualBox_014](https://user-images.githubusercontent.com/1457929/186910900-6118e89d-6baf-45de-b92f-a371bd8b6387.png)